core/src/main/java/discord4j/core/object/entity/Guild.java

```diff
@@ -2369,6 +2366,97 @@ public static NsfwLevel of(final int value) {
         }
     }
 
+    /**
+     * Describes the features of a guild.
+     */
+    public enum FeatureSet {
```

Comment:
i feel can add another field for declare if the feature is mutable this can be useful for the guild edit where you can pass features https://discord.com/developers/docs/resources/guild#guild-object-mutable-guild-features